src/shared/components/pointer-box/PointerBox.module.css

```diff
@@ -18,6 +21,15 @@ hence it is placed in top left corner of the screen with position: fixed */
 }
 
 .pointer {
+  fill: var(--_pointer-box-color);
```

Comment:
Curious why `pointer` have an underscore prepended

Reply:
I've declared this variable above (line 2) as:

```css
 --_pointer-box-color: var(--pointer-box-color, var(--color-black));
```

the underscore tries to communicate that this CSS variable is sort of private (of course nothing in CSS is private; but this is the intention). It is used only within this component (you will notice that this component uses it in two places). The variable that the component exposes outside and allows its parents to modify is a prettier looking `--pointer-box-color`. So, with one variable (`--_pointer-box-color`), we either capture the value set by the parent (`--pointer-box-color`), or the default value.

The alternative would be to repeat in multiple (in this component, two) places, the following line:

```
background-color: var(--pointer-box-color, var(--color-black));

...

fill: var(--pointer-box-color, var(--color-black));
```


This is the approach popularized by Lea Verou, e.g. [here](https://lea.verou.me/blog/2021/10/custom-properties-with-defaults/):

> My preferred solution is what I call pseudo-private custom properties. You use a different property internally than the one you expose, which is set to the one you expose plus the fallback: